aws_marketplace/using_model_packages/data_quality_monitoring/monitoring_data_quality_of_models.ipynb

```diff
@@ -0,0 +1,776 @@
+{
```

Comment:
<p>What baseline statistics is being computed? It makes the notebook more interesting if you add some plots demonstrating the baseline data distribution. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2024/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>